resources/assets/less/bem/mp-history-game.less

```diff
@@ -78,7 +92,15 @@
     margin: 0;
 
     &--title {
-      font-size: 20px;
+      font-size: 16px;
+      overflow: auto;
```

Comment:
a bit of mess...

![](https://s.myconan.net/2018-02/chrome_2018-02-02_20-23-33.png)